### ISSUE TITLE
Different Output

### ISSUE THREAD

**Issue by halsten** [shaded-enmity/armlet#2]
The output of your example differs in what your screenshot looks like. Plus you added some garbage text in the header `asdasfasgdgfdhfdh` as well, why? It's nice and I was looking for something like that, but am unsure of why also the left side of the main title bar is slightly wider than the box underneath it.

--halsten

**Comment by shaded-enmity** [comment]
@halsten Ups, sorry for that, but honestly the PGF/Tikz implementation was a mess which rendered improperly for every other instruction - after some experiments I realized that the rendering logic breaks down in a alot of cases and decided to rewrite just for web using JS/CSS - 

http://shaded-enmity.github.io/armlet/

http://shaded-enmity.github.io/armlet/insn.js
http://shaded-enmity.github.io/armlet/insn.css

Do you specifically need LaTeX?

**Comment by halsten** [comment]
Yeah, I was trying to look for something like that in LaTeX specifically. I was hoping you would have fixed it or something. Nonetheless, great work.
